app/models/version.rb

```diff
@@ -315,7 +315,11 @@ def authors_array
   end
 
   def sha256_hex
-    sha256.unpack("m0").first.unpack("H*").first if sha256
+    Version._sha256_hex(sha256) if sha256
+  end
+
+  def self._sha256_hex(raw)
```

Comment:
however one method is a class method and the other is an instance one. So I guess they could have the same name.